server/config.js

```diff
@@ -138,6 +138,7 @@ module.exports = {
     personalInformation:
       getEnv('ENABLE_PERSONAL_INFORMATION', 'false') === 'true',
     useRedisCache: getEnv('ENABLE_REDIS_CACHE', 'true') === 'true',
+    useMockAuth: getEnv('ENABLE_MOCK_AUTH', 'false') === 'true',
```

Comment:
Would it be worthwhile adding an explicit check for production here, so it refuses to mock auth in prod?

Reply:
Yeeah, will sort out some tests and see what I can do about doing that additional check 👍 Just wanted to get something out so we're not blocking writing E2E's for personalisation features

Reply:
I've added some tests to ensure the correct middleware gets mounted based on the feature toggle being set

Reply:
Hi @LucasCairns, did we update this to explicitly prevent mocked auth in production? It looks like the response to this comment might have been addressing a different point.

Reply:
I've set `ENABLE_MOCK_AUTH` to be explicitly `false` in our Helm templates. We don't have variable that says what environment we're in, but we have the `values` files for each environment. So we can split this out down the line if we choose to have mock auth in `Development` for example.